### PR TITLE
List antags in custody in round end screen

### LINKED ISSUE #18135: security needs # of antags in custody in end of round screen
this would incentivize security to play in a way that leaves syndicate members alive and in custody with them on the shuttle instead of killing every syndi

### PR DESCRIPTION
**Pull request by themias**
## About the PR

Lists the number of syndies in custody at end of the round. (cuffed on the shuttle)
Also adds a IN CUSTODY tag next to each arrested agent. (open to removing this part if it is too much)

Closes #18135

## Why / Balance

The idea was to give sec extra incentive to prioritize arresting instead of killing syndies.
This would also encourage them to remember to bring prisoners from the brig to the shuttle.

## Technical details

I've done some testing, and this also works for DC'd players and those who /ghost out of their bodies.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
screenshot

**Changelog**

:cl:
- add: The round end summary screen now lists the syndies in custody. (Cuffed on the shuttle)

### CODE REVIEW

**Comment by lzk228** [comment]
maybe put it in other line and make window little wider

**Comment by deltanedas** [comment]
wont this show for dragons too :trollface:

**Comment by UbaserB** [comment]
Would it be possible to add a "MIA" or something similar for dead syndicates, or "STRANDED" on syndicates that never left the station? Just ideas that could work too.

**Comment by Emisse** [comment]
I love how it looks

**Comment by themias** [comment]
> wont this show for dragons too :trollface:

I'll do some more testing later, but yea, it's only meant for those antags that have objectives, like traitor, nukie and ninja.
(While its rare to capture a nukie or ninja alive, it's not impossible.) I may need to update the code.

**Comment by themias** [comment]
> Would it be possible to add a "MIA" or something similar for dead syndicates, or "STRANDED" on syndicates that never left the station? Just ideas that could work too.

Stranded/escaped/etc. are all good ideas, I'm just worried the UI may become too crowded. I agree with TheDarkBox on discord that the round end screen probably needs a redesign, so I was hoping to keep the scope of this PR smaller for now.

**Comment by Emisse** [comment]
I do wonder if an "InCustody" component might be more prudent for future uses rather than storing it in objectives but thats out of scope

**Review by Emisse — APPROVED** [review]

File: Content.Server/Objectives/ObjectivesSystem.cs [hunk 1]
```diff
@@ -83,7 +83,10 @@ private void OnRoundEndText(RoundEndTextAppendEvent ev)
             }
 
             var result = Loc.GetString("objectives-round-end-result", ("count", total), ("agent", agent));
-            result += "\n" + Loc.GetString("objectives-round-end-result-in-custody", ("count", total), ("custody", totalInCustody), ("agent", agent));
+            if (agent == Loc.GetString("traitor-round-end-agent-name"))
```
Comment (deltanedas) [inline Content.Server/Objectives/ObjectivesSystem.cs:86]: yes its been merged but would you not just add ShowCustody to the event and have traitor set it to true

**Pull request merged**